src/store/getters.ts

```diff
@@ -82,6 +84,11 @@ const getters: GetterTree<VQBState, any> = {
    */
   pageno: (state: VQBState) =>
     state.dataset.paginationContext ? state.dataset.paginationContext.pageno : 1,
+  /**
+   * helper that is True if unique values are loading
+   */
+  isUniqueValuesLoading: (state: VQBState) => (column: string) =>
+    (state.dataset.headers.find(hdr => hdr.name === column) as DataSetColumn).isUniqueValuesLoading,
```

Comment:
I'm surprised that the explicit casting is necessary :thinking:

Reply:
The column names are not explicitly listed for typescript. Consequently, the `.find()` method may return an `undefined` value...